schemas/rules/operators/comparison/all.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/rules/operators/comparison/all.json",
+  "description": "Evaluates if all values are in an array, takes two arguments",
+  "type": "object",
+  "properties": {
+    "all": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "$ref": "https://eq.ons.gov.uk/rules/definitions.json#/array_value"
+      }
+    }
+  },
+  "required": ["all"],
```

Comment:
Initially thought `all` and `any` read fine but since most of the operators are very flexible and because our `all` and `any` only works to check `in`, I think we should make them explicit, so `all-in` and `any-in` or even our current naming of `contains-all` and `contains-any` otherwise I think it gets confusing as you can't do `all([True, True, True])`. I think of `all` and `any` like Python's all and any and JSON schemas `allOf` and `anyOf` which our one does not resemble. The new `and` and `or` structure would match `all` and `any` exactly from a technical view. Thoughts?

Reply:
I think using `contains` would be confusing, as it's the opposite of `in` - `[a,b] contains b` vs `b in [a,b]`, so my preference would be to go for `all-in` and `any-in`. 

> The new and and or structure would match all and any exactly from a technical view

I don't know what you mean by this.

Reply:
Ya fair point, therefore I'm happy to go with `all-in` and `any-in`.

> The new `and` and `or` structure would match all and any exactly from a technical view.

My bad, didn't word that right at all. All I meant by this is that our `and` and `or` operators pretty much does exactly what the proper `all()` and `any()` would do. i.e our JSON `and` is basically `and([True, True, False])` which is pretty much how `all` would work. So was just highlighting even more reason to be explicit by using `all-in` as we support `all` by supporting `and`. The same applies to `or` and `any`.

Reply:
Now I understand 🙂 

I will update to use `all-in` and `any-in`.